.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
can we make this less often? otherwise it's very noisy. I'm thinking monthly?

Reply:
service.go is practically untested :( except for some small bits and bobs

let's try it. if it's too noisy people will stop reviewing and we might need to readjust